digitalocean/resource_digitalocean_app_test.go

```diff
@@ -112,6 +112,22 @@ func TestAccDigitalOceanApp_Basic(t *testing.T) {
 						"digitalocean_app.foobar", "spec.0.database.0.engine", "PG"),
 				),
 			},
+			{
+				Config: fmt.Sprintf(testAccCheckDigitalOceanAppConfig_addImage, appName),
```

Comment:
Can we move this into it's own test function rather than it being an additional step here?

Then you can test it on its own without having to wait for all of the additional steps to complete. e.g.

    make testacc TESTARGS='-run=TestAccDigitalOceanApp_Image'